src/ai/backend/manager/api/exceptions.py

```diff
@@ -53,7 +52,7 @@ def __init__(self, extra_msg: Optional[str] = None, extra_data: Optional[Any] =
             body["msg"] = extra_msg
         if extra_data is not None:
             body["data"] = extra_data
-        self.body = json.dumps(body, cls=ExtendedJSONEncoder).encode()
+        self.body = dump_json(body)
```

Comment:
We have to cover this extended encoder. Do `orjson` support customized encoder?
Or we should serialize the body first.

Reply:
It works the same as before for `uuid`, `datetime` types in the `orjson` library.